metadetect/lsst/tests/test_lsst_metadetect.py

```diff
@@ -431,7 +431,7 @@ def test_lsst_metadetect_mfrac_ormask(show=False):
     rng = np.random.RandomState(seed=116)
 
     ntrial = 1
-    flag = 2**31
+    flag = 2**31-1
```

Comment:
A flag has to be a power of 2. So this is not the right fix.

Reply:
thanks @beckermr, you are right. I changed the flag to change a single bit